### PR TITLE
Add rarity and notes + some translation fixes

### PR DESCRIPTION
**Pull request by Quachero**

### CODE REVIEW

**Comment by EclipseTech** [comment]
Please fix the deleted file then I can merge this.

File: Properties/Strings.fr.resx [hunk 1]
```diff
@@ -205,7 +217,7 @@
     <value>Notes</value>
   </data>
   <data name="Freshwater Fish" xml:space="preserve">
-    <value>Poisson d’eau douce</value>
+    <value>Poisson d\'eau douce</value>
```
Comment (EclipseTech) [inline Properties/Strings.fr.resx:220]: XML says that only characters that need to be escaped are `&` and `<`

So this shouldn't need to be escaped.

**Review by EclipseTech — CHANGES_REQUESTED** [review]
Thanks for translating the rest.

**Comment by Quachero** [comment]
Hi

I remove backslash.

For border_x file, I know why git want to remove it. It's a case-sensitive problem. In your ref file, you have border_x.png and border_X.png. And don't know why, but the 2 files don't want to live together in my environnement ...

original_work

So, border_X.png is removed on my environnement, but isn't a really a problem, border-x.png is keeped :

my_merge_request

And on your code, you use only border_x.png, so border_X.png seem useless :

border_x_usage

**Comment by EclipseTech** [comment]
Thanks, I missed this somehow. Also didn't see that there were 2 border_x and border_X

**Pull request merged**